libsupport/src/JSONTracer.cpp

```diff
@@ -4,22 +4,23 @@
 
 #include <cstring>
 #include <iostream>
+#include <limits>
 #include <mutex>
 
 #include <arrow/memory_pool.h>
+#include <katana/Random.h>
```

Comment:
Revert this change please. Should be `"katana/Random.h"` since that's our convention for headers that we own.